package.json

```diff
@@ -18,30 +21,39 @@
     "@ckeditor/ckeditor5-heading": "Natim/ckeditor5-heading#short-heading",
     "@ckeditor/ckeditor5-list": "^1.0.0-alpha.2",
     "@ckeditor/ckeditor5-paragraph": "^1.0.0-alpha.2",
-    "babel-minify-webpack-plugin": "^0.2.0",
+    "classnames": "^2.2.5",
     "css-loader": "^0.28.7",
     "fxa-crypto-relier": "2.1.0",
     "jose-jwe-jws": "^0.1.5",
     "kinto": "^10.0.0",
     "kinto-http": "4.3.4",
     "material-design-lite": "1.3.0",
-    "node-sass": "^4.6.1",
     "quill": "1.2.6",
     "raw-loader": "^0.5.1",
-    "sass-loader": "^6.0.6",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
     "style-loader": "^0.19.0",
-    "testpilot-ga": "^0.2.1",
-    "webpack": "^3.8.1",
-    "webpack-sources": "1.0.1"
+    "testpilot-ga": "^0.2.1"
   },
   "devDependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-minify-webpack-plugin": "^0.2.0",
```

Comment:
do we need this module?

Reply:
It is used in ckeditor-build/webpack.config.js, so don't really know much about it but was not part of this PR. 
Should we create an issue to try simplifing ckeditor build process ?

Reply:
Ah yes, to avoid this in the future hopefully we can merge the two webpack files together :)